docs/requirements.rst

```diff
@@ -72,3 +74,19 @@ Additional Suggested Software
 
 - You may want to use autossh, available in various package management
   systems
+- If you are using systemd, sshuttle can notify it when the connection to
+  the remote end is established and the firewall rules are installed
+
+.. code-block:: ini
+   :emphasize-lines: 6
```

Comment:
Why do you want to emphasise line 6? Looks like that is the line with ``Type=notify``.